docs/features/logforwarding/outputs/otlp-forwarding.adoc

```diff
@@ -49,98 +49,93 @@ spec:
 * https://opentelemetry.io/docs/specs/semconv/[Semantic Conventions]
-* https://opentelemetry.io/docs/specs/otel/logs/data-model/[Logs Data Model]
+* https://opentelemetry.io/docs/specs/otel/logs/data-model/[OTel Logs Data Model]
 * https://opentelemetry.io/docs/specs/semconv/general/logs/[General Logs Attributes]
 
 === Data Mapping
-This document is not a complete spec and is mainly used as a guide.
+OpenShift Logging uses Vector Remap Language https://vector.dev/docs/reference/vrl/[(VRL)] to map and enrich the logging data.  The existing ViaQ fields are mapped to follow the semantic conventions of OpenTelemetry
 
-==== JSON Encoding
-OpenShift Logging uses Vector Remap Language https://vector.dev/docs/reference/vrl/[(VRL)] to map and enrich the logging data in order to the follow the semantic conventions of OpenTelemetry and its https://opentelemetry.io/docs/specs/otel/logs/data-model/[Logs Data Model]
+NOTE: This document and the following table is not a complete spec and is mainly used as a guide.  More work to refine the spec is ongoing.
 
-https://github.com/open-telemetry/opentelemetry-proto/blob/v1.3.1/examples/logs.json[Example JSON Encoded Logs] provide a visual representation of the data structure
-
-[%header,format=csv]
 |===
-ViaQ,OTEL
-@timestamp,                       timeUnixNano
-N/A,                            observedTimeUnixNano
-level,                            severityNumber
-message/structured,               body
---,--
-openshift.cluster_id,             k8s.cluster.uid
-hostname,                         k8s.node.name
-kubernetes.container_name,        k8s.container.name
-kubernetes.namespace_name,        k8s.namespace.name
-kubernetes.pod_name,              k8s.pod.name
-kubernetes.container_id,          k8s.container.id
-kubernetes.pod_id,                k8s.pod.uid
-kubernetes.namespace_labels.*,    k8s.namespace.label.*
-kubernetes.labels.*,              k8s.pod.label.*
-openshift.labels.*,               openshift.label.*
-log_type,                         openshift.log.type
-log_source,                       openshift.log.source
-hostname,                         url.domain
-requestURI (split),               url.path
-requestURI (split),               url.query
-responseStatus.code,              http.response.status.code
-verb,                             http.request.method_original
-userAgent,                        user_agent.original
-user.username,                    user.name
-systemd.u.SYSLOG_FACILITY,        syslog.facility
-systemd.u.SYSLOG_IDENTIFIER,      service.name
-systemd.t.COMM,                   process.command
-systemd.t.CMDLINE,                process.command_line
-systemd.t.EXE,                    process.executable.path
-systemd.t.GID,                    process.gid
-systemd.t.MACHINE_ID,             host.id
-systemd.t.PID,                    process.pid
-systemd.t.UID,                    process.user.id
+| ViaQ                         | OTel
```

Comment:
Should we drop this table all-to-gether given the OBS datamodel is supposed to be source of truth?  It also reduces the number of documents we need to maintain.

Reply:
removed table and section above